docs/core-concepts/core-sdk/cli/dotnet-new.md

```diff
@@ -0,0 +1,47 @@
+dotnet-new
+==========
+
+## NAME
+dotnet-new -- Create a new sample .NET Core project
+
+## SYNOPSIS
+dotnet-new [--type] [--lang]
+
+## DESCRIPTION
+The new command provides a convenient way to initalize a valid .NET Core project and sample source code to try out the CLI toolset. 
+
+This command is invoked in the context of a directory. When invoked, the command will result in two main artifacts being dropped to the directory: 
+
+1. A sample "Hello World" program that exists in `Program.cs` ( or `Program.fs` ) file.
+2. A valid `project.json` file
+
+> **Note:** As a workaround for packages not being on NuGet.org yet (since this is prelease software) the `dotnet-new`
+> command will also drop a `NuGet.config` file. This will be removed at RC2 release. 
```

Comment:
It seems likely this documentation will not go out until RC2 or very shortly before; do you have an issue tracking to come back and remove this line?